cloudinit/distros/netbsd.py

```diff
@@ -21,10 +21,18 @@ class NetBSD(cloudinit.distros.bsd.BSD):
     """
 
     ci_sudoers_fn = '/usr/pkg/etc/sudoers.d/90-cloud-init-users'
-
     group_add_cmd_prefix = ["groupadd"]
-    pkg_cmd_install_prefix = ["pkg_add", "-U"]
-    pkg_cmd_remove_prefix = ['pkg_delete']
+
+    def __init__(self, name, cfg, paths):
+        super().__init__(name, cfg, paths)
+        if os.path.exists("/usr/pkg/bin/pkgin"):
```

Comment:
Could we get a unit test that establishes that netbsd sets pkg_cmd_install_prefix and pkg_cmd_remove_prefix based on presence/absence of pkgin?

Also is the omission intentional for the else clause not setting pkg_cmd_update_prefix or pkg_cmd_upgrade_prefix?

Reply:
If this omission is intentional can you add a comment to the else clause about why pkg_cmd_upgrade/update are omitted in that case?

Reply:
Done